web/gui/dashboard_info.js

```diff
@@ -3049,43 +3049,78 @@ netdataDashboard.context = {
     // ------------------------------------------------------------------------
     // eBPF
 
+    'ebpf.tcp_functions': {
+        title : 'TCP calls',
+        info: 'TCP functions shows the number of calls for functions <code>tcp_sendmsg</code>, <code>tcp_cleanup_rbuf</code> and <code>tcp_close</code>, whether they return successfully or fail.'
```

Comment:
There is no need for phrases like `TCP functions shows` in every info string.

Reply:
Done!

Reply:
You are still repeating unneeded introduction phrases with the same meaning. The wrong pattern here is: `<chart> <does> <something>`. It should be `<something>` alone. Look at other info strings in `dashboard_info.js`.

My suggestion:

```
This chart shows the number of calls for...
--->
The number of calls for...
```
```
IO error shows the number of calls for...
--->
The number of calls for...
```
```
Process thread counts the number of times that...
--->
The number of times that...
```

and so on.